scripts/update-gh-pages.sh

```diff
@@ -8,17 +8,19 @@ echo "..."
 grunt build-gh-pages
 # Create temporary copy of index file
 cp index.html index-temp.html
+cp node_modules/bootstrap/dist/css/bootstrap.min.css temp/bootstrap.min.css
```

Comment:
could you remove this step to be after line 15? i think it makes more sense there